gettext-setup.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 1.9.2'
 
-  spec.add_dependency 'fast_gettext', '~> 1.1.0'
+  spec.add_dependency 'fast_gettext', '~> 1.7'
```

Comment:
I guess `~> 1.1` would be enough, allowing both 1.1.x and 1.7+.

I was just about to propose this change, with basically the same idea behind my head regarding the FreeBSD ports of Puppet, see:
https://github.com/puppetlabs/puppet/pull/7406

Reply:
Yes I think `~> 1.1` would work for us everywhere.

Reply:
Well, may be yes

Reply:
@nigelkersten what is next?

Reply:
@majioa you can amend this commit to depend on `~> 1.1` and then rebase your branch on top of master to get rid of the conflicts.  You will then need to force push and the PR should be in good shape.

Reply:
updated